src/plugins/specload/testapp.c

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file
+ *
+ * @brief Tests for specload plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+#include <kdbinvoke.h>
+#include <kdbmodule.h>
+
+#include "testdata.h"
+
+// keep #ifdef in sync with kdb export
+#ifdef _WIN32
+#define STDOUT_FILENAME ("CON")
+#else
+#define STDOUT_FILENAME ("/dev/stdout")
+#endif
+
+static int outputKeySet (KeySet * ks)
+{
+	Key * errorKey = keyNew (0, KEY_END);
+
+	KeySet * modules = ksNew (0, KS_END);
+	elektraModulesInit (modules, 0);
```

Comment:
Why is elektraModulesInit needed? That should be done by elektraInvokeOpen

Reply:
Shouldn't really matter considering what `elektraModulesInit` actually does...

But I will remove it anyway

Reply:
You are right, it most likely will not be related to the problem we have here.